elements/rh-cta/src/rh-cta.scss

```diff
@@ -72,7 +72,7 @@ $rh-cta__arrow-size:                        0.313em;
     padding: calc(#{rh-var(container-padding)} * 0.5) calc(#{rh-var(container-padding)} * 2);
     border-radius: calc(#{rh-var(ui--border-radius)} * 20);
     border: #{rh-var(ui--border-width)} #{rh-var(ui--border-style)} transparent;
-    text-decoration: none;
```

Comment:
@markcaron I'd love to avoid using !important in situations like these.  It makes it difficult to override when that is desirable.